### ISSUE TITLE
Improve Orientation around Braille panel

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#20154]
### Your idea

Currently when using arrow keys to move up/down staves/parts in the Braille panel it is hard to know which one you are on.  This could become particularly confusing with scores with more than a couple of instrument parts.
This could be fixed by adding screen reader announcements of part/instrument names as you move up and down just as in the score.
But also for those who may use Braille only and turn off screen reading could there be an instrument/part name added in Braille abbreviation at the start of each Braille line.

### Problem to be solved

This would remove confusion for blind users as to which part/instrument of Braille they are viewing at any given time.

**Comment by jmwmusic** [comment]
I forgot to mention bar numbers at the start of each bar in the Braille panel would greatly help orientation.  Basically both vertically and horizontally in the score currently it is really confusing to know where you are.

**Comment by msviolafangirl** [comment]
I agree with everything you said. I think the challenge is to be able to meaningfully fit all the information on the braille display in a way that doesn't consume too much space.
